idea/testData/quickfix/variables/changeMutability/funParameter.kt

```diff
@@ -4,4 +4,5 @@
 // ERROR: Val cannot be reassigned
 fun fun1(i: Int) {
     <caret>i = 2
-}
\ No newline at end of file
+}
+/* FIR_COMPARISON */
```

Comment:
Just to make sure. Is this something like `// FIR_IDENTICAL`?

Unlike any other test data changes, this one doesn't have overriding property/value parameter. So, I'm wondering why this directive is added. I can see `// "Change to var" "false"` (and all others have `"true"`) So, I guess this means the test checks that quick fix doesn't show up?

Reply:
`FIR_COMPARISON` directive means that corresponding test (with testate from FE10 plugin) should work in IDEA FIR plugin. If testdata file do not have this directive then the test will be ignored, see https://github.com/JetBrains/kotlin/blob/d72a2d39da2105939c5c300023eaf1fd172be4d9/idea/idea-fir/tests/org/jetbrains/kotlin/idea/quickfix/AbstractHighLevelQuickFixTest.kt#L15

Reply:
@darthorimar Why can't we re-use the `// FIR_IDENTICAL` directive? Why do we need a different `/* FIR_COMPARISON */` directive?

@jsjeon That is my understanding as well (verify "Change to var" fix doesn't show up). Also the `// ACTION` and `// ERROR` lines should verify that those actions and errors show up with no extra actions and errors, however these are currently disabled: https://github.com/JetBrains/kotlin/blob/d72a2d39da2105939c5c300023eaf1fd172be4d9/idea/idea-fir/tests/org/jetbrains/kotlin/idea/quickfix/AbstractHighLevelQuickFixTest.kt#L25-L26 @darthorimar Is that correct?

Reply:
We can replace, but FIR_COMPRATION is already used for a lot of idea fir tests, so let‘s keep that as is

Reply:
//action and //errors directives are disabled because we have a different set of diagnostics in fir and fe1.0. Maybe we should just have additional // fir_errors and //fir_actions instead 🤔

Reply:
Sorry I meant `// FIR_COMPARISON` but okay we can keep as-is